settings/blt.settings.php

```diff
@@ -83,7 +83,7 @@
 /**
  * Acquia Cloud settings.
  */
-if ($is_ah_env && file_exists('/var/www/site-php')) {
+if ($is_ah_env && file_exists('/var/www/site-php' && !$is_acsf)) {
```

Comment:
I can submit a PR, just wanted to make sure it wasn't intentional. I assume this works on ACSF b/c the `file_exists()` command returns false (maybe?), but this would then not run correctly on ACE, I believe.